src/fhir-tool/Program.cs

```diff
@@ -607,17 +603,23 @@ private static Questionnaire.ItemComponent CreateItemComponentV2(QuestionnaireIt
             Questionnaire.ItemComponent itemComponent = new Questionnaire.ItemComponent
             {
                 Type = itemType,
-                LinkId = item.LinkId,
-                Prefix = string.IsNullOrEmpty(item.Prefix) ? null : item.Prefix,
-                Text = string.IsNullOrEmpty(item.Text) ? null : item.Text
             };
+            if (!string.IsNullOrEmpty(item.LinkId))
```

Comment:
Code is getting somewhat verbose with all the if's.
Would it be possible to retain the more terse syntax:

Property = string.IsNullOrEmpty(item.Property) ? null : item.Property

or does assigning null to properties have unwanted side effects?

Reply:
I agree that it gets quite verbose, but unwanted side effects is the reason currently